package/MDAnalysis/core/universe.py

```diff
@@ -850,6 +850,15 @@ def add_TopologyAttr(self, topologyattr, values=None):
                     n_residues=self._topology.n_residues,
                     n_segments=self._topology.n_segments,
                     values=values)
+        alias_pairs = [("bfactors", "tempfactors"),
```

Comment:
Can we add this as note in the `add_TopologyAttr` docstring? That way folks understand what is going on with regards to this.

Reply:
Added a versionchanged.